modules/swagger-codegen/src/main/resources/typescript-fetch/api.mustache

```diff
@@ -0,0 +1,309 @@
+// tslint:disable
+{{>licenseInfo}}
+
+import * as url                                 	    from "url";
+import * as isomorphicFetch                     	    from "isomorphic-fetch";
```

Comment:
Yes, sure, that's why it's called isomorphic. My point is, does it worth coupling to a specific dependency? Why not inject it from the outside, based on the declared interface?